tests/integration/testlib/iprule.py

```diff
@@ -36,34 +36,29 @@ def ip_rule_exist_in_os(ip_from, ip_to, priority, table):
     logging.debug(f'Current ip rules in OS: {result[1]}')
     assert result[0] == 0
     current_rules = json.loads(result[1])
-    found = True
     for rule in current_rules:
+        found = False
```

Comment:
IMO having a rule matcher function that returns the matched rule would be nicer and avoid this ugly flag.
```
rule = lookup_rule(ip_from, ip_to, priority, table, current_rules)
assert rule
```